index.html

```diff
@@ -14,6 +14,88 @@
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Karma</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    </head>
+    
+    <!---Body-->
+    <body>
+    <header>
+    <!-- Nav bar -->
+    <img class="logo" src="img/karma-logo.svg" alt="coffee icon" />
+    <nav>
+    <ul class="nav__links" >
+    <li><b><a class="a-bold-1" href="#">Meet Karma</a></b></li>
+    <li><a href="#">How it works</a></li>
+    <li><a href="#">Store</a></li>
+    <li><a href="#">Blog</a></li>
+    <li><a href="#">Help</a></li>
+    <li><a href="#">Login</a></li>
+    </ul>
+    </nav>
+    </header>
+    
+   <!-- Main -->
+   <main>
+   <!-- Section -->
+   <section>
+    <div>
+   <div class="image__container">
+   <div class="inner__container">
+   <h1>Introducing Karma</h1>
+   <p>Bring WiFi with you, everywhere you go.</p>
+   <a class="btn" href="#">Learn More</a>
+   </div>
+  </div>
+   </section>
+   <section>
+   <div>
+    <h2>Everyone needs a little Karma</h2>
+    <div class="image__center">
+    <div>
+    <img src="img/icon-devices.svg" alt="Devices image" />
+    <p>Internet for all devices</p>
+    </div>
+    <div>
+    <img src="img/icon-coffee.svg" alt="Devices image" />
+    <p>Boost your productivity</p>
+    </div>
+    <div>
+    <img src="img/icon-refill.svg" alt="Devices image" />
+    <p>Pay as you go</p>
+    </div>
+  </div>
+</div>
+    </section>
+    </main>
 
+    <br />
+    <hr />
+    <!-- Footer -->
+    <footer>
+    <div>
+    <h4>Join us on</h4>
```

Comment:
The use of h4 here is incorrect.

If this is a heading then you should use h2, as headings are about hierarchy not size, and the only other heading you have on your page is h1, so the next heading should be h2 and not jump to h4, and then assign in css the font size you wish for the h2.

If this is not a heading and you used h4 to get the sizing you want then you should just have used a `<p>` tag and then allocated the font size you want for it.